scripts/lint/donotmerge-git-commits.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env sh
+
+DO_NOT_MERGE_KEYWORD="donotmerge"
+DEFAULT_BASE_BRANCH="develop"
+BASE_BRANCH="${CHANGE_TARGET:-$DEFAULT_BASE_BRANCH}"
+FEATURE_BRANCH="${CHANGE_BRANCH:-$(git branch --show-current)}"
+
+if [[ -n "$CI" && "$CI" == "true" ]]; then
+    # Configuring remote origin for CI to resolve origin branches when comparing commits
+    git config remote.origin.fetch '+refs/heads/*:refs/remotes/origin/*'
+    git fetch origin "${BASE_BRANCH}:${BASE_BRANCH}"
+    COMMITS=$(git log "${BASE_BRANCH}"..HEAD --oneline)
```

Comment:
Just quote things as one string, since it's one argument:
```suggestion
    COMMITS=$(git log "${BASE_BRANCH}..HEAD" --oneline)
```